docs/_interface/Private-Network.md

```diff
@@ -3,103 +3,84 @@ title: Private Networks
 sort_key: B
 ---
 
-This guide explains how to set up a private network of multiple Geth nodes. An Ethereum
-network is a private network if the nodes are not connected to the main network. In this
-context private only means reserved or isolated, rather than protected or secure.
+This guide explains how to set up a private network of multiple Geth nodes. An Ethereum network is private if the nodes are not connected to the main network. In this
+context private only means reserved or isolated, rather than protected or secure. A fully controlled, private Ethereum network is useful as a backend for core developers working on issues relating to networking/blockchain syncing etc. Private networks are also useful for Dapp developers testing multi-block and multi-user scenarios. 
+
+This page describes the setup process and configuration options required to establish a private network. An end-to-end tutorial for setting up a simple private network is available in the [final section below](#end-to-end-example).
+
+
+## Prerequisites
+
+To follow the tutorial on this page it is necessary to have a working Geth installation (instructions [here](../../_site/docs/install-and-build/installing-geth.html)). It is also helpful to understand Geth fundamentals (see [Getting Started](../../docs/_getting-started/index.md)).
+
+
+## Private Networks
+
+A private network is composed of multiple Ethereum nodes that are all run on the local machine. In order to run multiple nodes locally, each one requires a separate data directory (`--datadir`), a different `eth` and `http` port and a unique `ipc` endpoint. The nodes must also know about each other and be able to exchange information, share an initial state and a common consensus algorithm. The remainder of this page will explain how to configure Geth so that these basic requirements are met, enabling a private network to be started.
```

Comment:
> A private network is composed of multiple Ethereum nodes that are all run on the local machine.

A private network does not need to be run all off one local machine. It's pretty common to run a 'private network' across a set of hosts, e.g. for hackathons

Reply:
> a different `eth` and `http` port and a unique `ipc` endpoint.

The `eth` is simply `--port`. And re ipc endpoint, doesn' that follow along with the datadir? So if you set the datadir, the ipc will be also be changed accordingly

Reply:
yes, that is right. This sentence was inherited from the original doc, I thought it was useful to point out which specific things should differ between the nodes, even if they don't have to be explicitly defined. But on reflection it is a bit confusing - will change.